### DISCUSSION TITLE
Tag chosen but disappears

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#134004] [Mobile]
On https://github.com/QuantumBadger/RedReader/issues/1213 I saw that I could pick a tag for the issue I was creating. 

Okay well after saving the issue, I don't see the tag that I picked. 

Perhaps only administrators are allowed to pick tags and not regular users who don't own the repository. 

Then why are tags shown as that something you could pick anyway? 

That's not how it looks on desktop when entering the issue.

**Comment by std-enigma** [comment]
Yeah regular users don't have the permission to put labels on their issues

**Reply by jidanni** [reply]
I see. Then then giving them the choice is a bug!

**Comment by ettaboyle** [comment]
@jidanni was this something you experienced only on mobile, but not on the desktop? Have you tried to replicate the issue on desktop?
I'm curious if this is mobile specific, or if its related to issues and should be recategorized.

**Reply by jidanni** [reply]
On desktop, me, a mere regular user, is not offered a tag choice.

Try it yourself. On the bottom of the screen on mobile, there are tags you can choose, that should not be there.
